### PR TITLE
fix: 🐛 `next` field for ResultSet when size is not provided

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Use result size, not requested size when calculating next page

### Breaking Changes

None

### JIRA Link

✅ Closes: [DA-502](https://polymesh.atlassian.net/browse/DA-502)

### Checklist

- [ ] Updated the Readme.md (if required) ?

[DA-502]: https://polymesh.atlassian.net/browse/DA-502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
   testMatch: ['**/__tests__/**/*.(ts|tsx)'],
   testPathIgnorePatterns: ['dist'],
+  modulePathIgnorePatterns: ['dist'],
```
Comment (polymath-eric) [inline jest.config.js:8]: resolves a jest "haste-map" warning on newer node versions

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**